frontend/src/api/visitors/bounceRateService.js

```diff
@@ -39,7 +39,7 @@ const fetchChartValues = (startDate, endDate, interval) => {
 };
 
 const fetchTableValues = (startDate, endDate, groupBy) => {
-    return requestService.get(resourceUrl + '/table-visitors/count-total', {}, {
+    return requestService.get(resourceUrl + '/table-visitors/bounce-rate', {}, {
```

Comment:
you changed route here, but did not in api.php, is it correct?

Reply:
yes, someone set wrong route here and it is right route